src/main/java/org/opentripplanner/api/resource/TimeGridWs.java

```diff
@@ -109,7 +107,8 @@ public Response getTimeGridPng(@QueryParam("base64") @DefaultValue("false") bool
                     .getCoordinate();
 
         // Get a sample grid
-        ZSampleGrid<WTWD> sampleGrid = sampleGridRenderer.getSampleGrid(tgRequest, sptRequest);
+		ZSampleGrid<WTWD> sampleGrid = router.sampleGridRenderer.getSampleGrid(
```

Comment:
Your editor appears to be inserting tabs here and causing different line breaking.